Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -519,6 +519,8 @@ def audit_deps
   end
 
   def audit_conflicts
+    conflicts_whitelist = %w[node@0.12 node@0.10]
```

Comment:
Sorry, I mean move the addition down inside the `if formula.conflicts.any? && formula.versioned_formula?` block as I think the other conflicts changes should be done. Alternatively, move this whole lot under the loop below.